Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,169 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
     }
 }
 
+void
+WarpX::PrintMainPICparameters ()
+{
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n";
+    amrex::Print()<<"--------------------------- MAIN EM PIC PARAMETERS ----------------------------"<<"\n";
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n";
+
+    // Print geometry dimensionality
+    amrex::ParmParse pp_geometry("geometry");
+    std::string dim;
+    pp_geometry.query( "dims", dim );
+    if (dim=="2") {
+      amrex::Print()<< "Geometry:             | 2 (XZ)\n";
+    }
+    else {
+      amrex::Print()<< "Geometry:             |" <<dim<<"\n";
+    }
+
+    #ifdef WARPX_DIM_RZ
+      amrex::Print()<< "                      | - n_rz_azimuthal_modes=" <<WarpX::n_rz_azimuthal_modes<<"\n";
+    #endif // WARPX_USE_RZ
+    //Print solver's operation mode (e.g., EM or electrostatic)
+    if (do_electrostatic == ElectrostaticSolverAlgo::LabFrame) {
+      amrex::Print()<< "Operation mode:       | Electrostatic\n";
+      amrex::Print()<< "                      | - labframe\n";
```

Comment:
How about "laboratory frame" or something like this (more verbose):
```suggestion
      amrex::Print()<< "                      | - laboratory frame\n";
```